pkgs/development/python-modules/i3ipc/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, buildPythonPackage, enum-compat }:
+
+let
+  version = "1.3.0";
+  sha256 = "1rw9mq18np6bf8xp8vvc21bi5q8xjmj8dck2vsa1hy8bk434259m";
+
+in buildPythonPackage rec {
+  pname = "i3ipc";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "acrisci";
+    repo = "i3ipc-python";
+    rev = "refs/tags/v${version}";
+    inherit sha256;
+    fetchSubmodules = true; # Fetching by tag does not work otherwise
```

Comment:
seems to fetch just fine here without. Note also that `refs/tags` can be dropped.